ddl/serial_test.go

```diff
@@ -186,10 +198,7 @@ func (s *testSerialSuite) TestMultiRegionGetTableEndHandle(c *C) {
 	testCtx := newTestMaxTableRowIDContext(c, d, tbl)
 
 	// Split the table.
-	cluster := mocktikv.NewCluster()
-	mvccStore := mocktikv.MustNewMVCCStore()
-	defer mvccStore.Close()
-	cluster.SplitTable(mvccStore, tblID, 100)
+	s.cluster.SplitTable(tblID, 100)
```

Comment:
This a fix of an *incorrect fix* introduced by https://github.com/pingcap/tidb/pull/14592